docs/charter.md

```diff
@@ -147,58 +150,52 @@ This means that the products of our work should be considered _best effort_, and
 
 ### Joining the _CPANSec_
 
-The _CPANSec_ forum itself is moderated and for invited volunteers. To join it, please reach out to our mailing list. (**FIXME**). 
-
-
-## Example Means and Methods
-
-The _CPANSec_ activities may include a number of efforts, ongoing activities, security projects, tooling development and managing of publication channels and other resources.
-
-Examples of these may include…
-
-
-### Ongoing activities
-
-- Gathering of security-relevant statistics, metrics and metadata
-- Publish CPAN advisories
-- Apply for CVE numbers
-- …
+The _CPANSec_ forum itself is moderated and for invited volunteers. To join it, please reach out to our mailing list, or our IRC chat for an informal conversation. (**FIXME**). 
```

Comment:
Shouldn't it be enough to link this section to the "public contact points" section below?

Reply:
I was actually thinking that we'd put together a more complete set of instructions for joining CPANSec, including reasonable requirements for establishing an internal trust regime...

Reply:
But sure, we can refer to the contact points. I'll add that.